patches/api/0374-Added-EntityToggleSitEvent.patch

```diff
@@ -0,0 +1,78 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: KyGuy2002 <IEatBeans#1165>
+Date: Fri, 11 Mar 2022 15:33:10 +0000
+Subject: [PATCH] Added EntityToggleSitEvent
+
+
+diff --git a/src/main/java/io/papermc/paper/event/entity/EntityToggleSitEvent.java b/src/main/java/io/papermc/paper/event/entity/EntityToggleSitEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..76f01788cd3b6249dfdcf0b5395d25343935bcfb
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/event/entity/EntityToggleSitEvent.java
+@@ -0,0 +1,66 @@
++package io.papermc.paper.event.entity;
++
++import org.bukkit.entity.Entity;
++import org.bukkit.event.Cancellable;
++import org.bukkit.event.HandlerList;
++import org.bukkit.event.entity.EntityEvent;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * Is called when a Panda or an entity that extends TamableAnimal changes its sitting state.
```

Comment:
`TamableAnimal` class is not a part of Paper API - I don't think it's a good idea to refer it here.

Reply:
True, what do you think a better wording would be, just “any entity that can sit”

Reply:
Well, I think the first wording was just fine? `Sittable` in Paper API, while there's no equivalent for it in NMS, does cover cats, wolves, parrots, foxes and pandas.